src/Apps/Collect/Components/Filters/SortFilter.tsx

```diff
@@ -7,32 +7,55 @@ import { Button, FilterIcon, Flex, SelectSmall, Spacer } from "@artsy/palette"
 export const SortFilter: React.FC<{
   filters: FilterState
   onShow?: () => void
-}> = ({ filters, onShow }) => {
+  isCollection?: boolean
+}> = ({ filters, onShow, isCollection = false }) => {
+  let options = [
+    {
+      value: "-decayed_merch",
+      text: "Default",
+    },
+  ]
+
+  const include_for_collections = [
```

Comment:
This looks like ruby case :) We should use `camelCase`, per other idioms in the codebase.